pyproject.toml

```diff
@@ -132,3 +132,7 @@ ban-relative-imports = "all"
   # Allow print
   "T201",
 ]
+
+[tool.isort]
+profile = 'black'
+skip = 'migrations'
```

Comment:
We use Ruff now for import sorting. If we're still using isort somewhere we should remove that.